vendor/bat-native-ledger/src/bat/ledger/internal/media/vimeo.cc

```diff
@@ -7,6 +7,8 @@
 #include <cmath>
 #include <utility>
 #include <vector>
+#include <regex>
```

Comment:
are this two includes needed? I don't see any code that would use it

Reply:
Again good catch - my first approach used regex, and iostream was a hack to do debug logging - so neither are needed.

Reply:
Fixed